content/docs/docs-nav.yaml

```
@@ -63,3 +63,4 @@ nav:
     - Required annotations: '/docs/details/required-annotations/'
     - Previewing config changes: '/docs/details/previewing-changes/'
     - Removing Components: '/docs/details/unregister-components/'
+    - Accessing Public resources from the Backend: '/docs/details/backend-reading-allow-list/'
```

```suggestion
    - Accessing Public resources: '/docs/details/backend-reading-allow-list/'
```

This title is too long for the sidebar and wraps in a weird way. The shorter version should fit better.